reference/features/README.md

```diff
@@ -6,7 +6,7 @@ Understanding our reference documentation is easier with background knowledge of
 
 ## Table of contents
 
-[Authentication](/reference/features/authentication.md)
+[Security](/reference/features/authentication.md)
```

Comment:
```suggestion
[Security](/reference/features/security.md)
```

Reply:
You updated all existing the links in your PR, so I didn’t update them here.